bin/run-migration-job.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python
+"""
+This script will either migrate or rollback the swoop database to a specified
+migration version. The migration version, action type (of either migrate/rollback),
+and an override parameter are all specified in the values.yml from the Swoop-DB helm
+chart.
+
+It requires the following environment variables be set:
+
+    - PGHOST:
+          Postgres K8s service name which will be used to connect to the database
+    - PGUSER:
+          name of the user (role) that will perform the migrations
+    - PGPORT:
+          port number of the database container
+    - PGDATABASE: name of the database which will be used for migrations
+    - Any additional libpq-supported connection parameters
+          (https://www.postgresql.org/docs/current/libpq-envars.html)
+"""
+import argparse
+import asyncio
+import os
+import sys
+
+from buildpg import V, funcs, render
+
+from swoop.db import SwoopDB
+
+APPLICATION_ROLES = ["user_api", "user_caboose", "user_conductor"]
+
+
+def stderr(*args, **kwargs) -> None:
+    kwargs["file"] = sys.stderr
+    print(*args, **kwargs)
+
+
+def check_positive(value) -> int:
+    ivalue = int(value)
+    if ivalue <= 0:
+        raise argparse.ArgumentTypeError(
+            "%s is an invalid migration version number. Only \
+                positive values are supported."
+            % value
+        )
+    return ivalue
+
+
+async def run_migrations() -> None:
+    dbname: str = os.environ["PGDATABASE"]
+
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument(
+        "--action",
+        choices=["migrate", "rollback"],
+        help="An action to take, either migrate or rollback",
+    )
```

Comment:
Again, this is a flag with a sane default to migrate--rollbacks are an uncommon occurrence. So how about:

```suggestion
    parser.add_argument(
        "--rollback",
        action="store_false",
        help="Rollback instead of migrating forward. Default false.",
    )
```

Reply:
I think this should be 'default true' since `store_false` creates a default value of True if the argument is not provided.

Reply:
Also, we want to rollback if the rollback argument is provided, so I think this should be `store_true` instead of `store_false`... and in that case, the default in the help message can be kept as false.